integration_test/agent_metrics/metadata.yaml

```diff
@@ -0,0 +1,212 @@
+expected_metrics:
```

Comment:
This file seems not include a complete list of all agent metrics we support in: https://cloud.google.com/monitoring/api/metrics_opsagent#agent. This file honestly converts what we had long in metricsForPlatform into this yaml format.
 

For agent/agent, it's missing uptime

For agent/cpu, it's missing usage.. i haven't checked the rest.

The bug was originated from b/209686852 which created by me when one of our released ops-agent started missing agent metrics as we never included it in any tests. Thus it's not entirely a duplicate of b/230790404 which seems only focusing on having check the details of each metrics(which we also must have tests cover that IMO)

@qingling128  for opinion we shall have a separate PR/Bug to cover all agent prefixed metrics. I've just de-deplicate and reopened b/209686852

Reply:
Case by case, we knew that api_request_count was missed because client found out that and asked why the metric stopped ingesting after upgrading. That issue had been fixed long ago and the test was added to ensure such metric shall not be missed. But for all other metrics that ain't included in metricsForPlatform. They might be missing starting the very beginning thus no one found that out. Or best to be, they are always there. Adding them to the tests shall not fail the test

Reply:
Sync'ed offline, the missing metrics (comment i made yesterday could be done in a separate PR in b/209686852)